frontend/src/pages/ApartmentPage.tsx

```diff
@@ -184,13 +203,30 @@ const ApartmentPage = (): ReactElement => {
 
   const Header = (
     <>
-      <Grid container item spacing={3} justify="space-between" alignItems="center">
-        <Grid item>
-          <Typography variant="h4">Reviews ({reviewData.length})</Typography>
-          {reviewData.length === 0 && (
-            <Typography>No reviews available. Be the first to leave one!</Typography>
+      <Grid container alignItems="center">
```

Comment:
Based on new designs, I changed it to look like this
<img width="835" alt="Screen Shot 2022-04-21 at 8 55 58 PM" src="https://user-images.githubusercontent.com/57203639/164580892-afd11d58-8475-4a65-b867-ae7592f76ccf.png">

Here's a video showing resizing: 

https://user-images.githubusercontent.com/57203639/164580926-953cf0cf-6844-47e7-bced-1c91dec9a2ad.mov